docs/src/api/class-browsertype.md

```diff
@@ -153,7 +153,7 @@ Firefox user preferences. Learn more about the Firefox user preferences at
 
 ### option: BrowserType.launch.firefoxUserPrefs
 * langs: csharp, java
-- `firefoxUserPrefs` <[Object]<[string], [any]>>
```

Comment:
I'd keep it `any` to highlight that it can also be a primitive type, we also use `any` in other places in the docs so you need support for that anyway. Does it cause any problems?

Reply:
Sure, I can flip the handling around. The current logic there was, that I'd translate that to a generic `<T>`, so this caused problems. But even if I do actually define that return as `object` and add an extension method down the line that is generic, it'll be fine.